docs/meshstack.openstack.index.md

```diff
@@ -43,37 +40,31 @@ meshStack also supports the following OpenStack APIs for advanced features
-meshStack expects you to configure [Federated Identity](https://docs.openstack.org/keystone/ussuri/admin/federation/federated_identity.html) so that
-your identity provider stores the `euid` value in the `User.federated[].protocols[].unique_id` field of the [Keystone User object](https://docs.openstack.org/api-ref/identity/v3/?expanded=list-users-detail,show-user-details-detail#show-user-details).
-
-When configuring the meshPlatform in meshStack's dhall config model, provide the `idp_id` as `idp-provider` and `protocol_id` as `idp-protocol`
-property respectively
-
-```dhall
-...
-let platformConfig =
-      Platform.OpenStack::{
-      ...
-      , idp-provider = "aad"
-      , idp-protocol = "openid"
-      }
-...
-```
+meshStack expects that your OpenStack uses [Federated Identity](https://docs.openstack.org/keystone/2023.2/admin/federation/federated_identity.html). 
+
+You should set up OpenStack Keystone so that your identity provider stores the `euid` value in the `User.name` field of the [Keystone User object](https://docs.openstack.org/api-ref/identity/v3/?expanded=list-users-detail,show-user-details-detail#show-user-details). 
+
+## Integration Configuration
+
+### meshStack Admin Accounts
+
+meshStack requires two admin user accounts for integration with OpenStack. 
+
+- the `replicator` admin account must have permission to create projects, groups and assign roles. This typically requires an `admin` role on the admin project, unless your OpenStack deployment has additional fine-grained policies available.
+- the `metering` account must have `reader` role access across all OpenStack projects. This user is used to collect [metering data](./meshstack.openstack.metering.md) for all projects from OpenStack services.
+
+### Permission Replication
 
 During replication, meshStack will make sure that users have access to the OpenStack projects they are assigned to in meshStack.
-Users will be assigned to the according groups (per meshProject) that meshStack creates in OpenStack.
-If meshStack is about to assign a user that does not exist in OpenStack yet, meshStack will create this user with the
-according IDP attributes and create the user in the Keystone domain `MeshUsers`.
+If meshStack finds that a user does not exist in OpenStack yet, meshStack will log a replication remark and skip assigning that user's permission. 
+When the user is created in OpenStack at any later time, meshStack will pick up this user to assign him to the according groups in the next replication.
 
-That way once the user logs in via the IdP, he will be mapped to the user created by meshStack.
-If the user already exists in OpenStack during replication, meshStack will pick up this user to assign him to the according groups.
+Users will be assigned to the according groups (per meshProject) that meshStack creates in OpenStack.
```

Comment:
```suggestion
Users will be assigned to the according groups (per project in meshStack) that meshStack creates in OpenStack.
```
I think if you do really want to differentiate between a OS project and a meshStack project this is a good pattern.

Reply:
Yeah, dropping the mesh* prefix requires becoming a bit more verbose in those docs.